Tasks/Common/docker-common/dockercommandutils.ts

```diff
@@ -89,6 +91,38 @@ export async function getLayers(connection: ContainerConnection, imageId: string
     return layers.reverse();
 }
 
+export function getImageSize(layers: { [key: string]: string }[]): string {
+    let imageSize = 0;
+    for (const layer of layers) {
+        for (let key in layer) {
+            if (key.toLowerCase() === "size") {
+                const layerSize = extractSizeInBytes(layer[key]);
+                imageSize += layerSize;
+            }
+        }
+    }
+
+    return imageSize.toString() + "B";
+}
+
+export function extractSizeInBytes(size: string): number {
+    const sizeStringValue = size.match(matchPatternForSize);
+    if (sizeStringValue && sizeStringValue.length > 0) {
+        const sizeIntValue = parseFloat(sizeStringValue[0]);
+        const sizeUnit = size.substring(sizeIntValue.toString().length);
+        switch (sizeUnit.toLowerCase()) {
+            case "b": return sizeIntValue;
```

Comment:
i would recommend them like, 1K = 1024, 1m like 1k*1024 something like that a const